src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -204,40 +220,42 @@ export function EmbeddedPositionSearchPanel({
         const defSlotProps = {
             searchSummary: {
                 title: undefined,
-                searchType, coords, radius, fileName, rows,
                 showCoords: false,
                 showRadius: false,
                 showFileInfo: false,
                 sx:{}
             },
         };
-        
+
         return (
             isOpen ?
                 <div/>:
-                <Stack width={'100%'} >
+                <Stack p={0}>
                     <FormPanel
                         onSuccess={(request) => doSearch(request)}
                         direction={'row'}
                         width={'100%'}
                         slotProps={{
-                            searchBar: {px:0, py:0, justifyContent: 'left'},
+                            searchBar: {p:0, justifyContent: 'right'},
                         }}
                         cancelText=''>
-                        <Slot component={SearchSummary} {...defSlotProps.searchSummary} slotProps={slotProps?.searchSummary}/>
+                        <Stack {...{width:'100%', alignItems:'center'}}>
+                            <Slot component={SearchSummary} {...defSlotProps.searchSummary} searchType={searchType}
+                                  coords={coords} radius={radius} fileName={fileName}
+                                  rows={rows} slotProps={slotProps?.searchSummary}/>
+                        </Stack>
                     </FormPanel>
             </Stack>
         );
     };
 
     return (
-        <div key='targetGroup'
-             onMouseDown={() => {
-                      if (!isSearchPanel) setIsHovered(false);
-                      else setIsHovered(true);
-             }}
-            style={{display: 'flex', flexDirection: 'column', alignItems: 'center', alignSelf: 'stretch', height:'100%',
-                         paddingBottom:insetSpacial?0:20, position: 'relative'}}>
+        <Stack key='targetGroup' alignItems='center' height='100%' paddingBottom={insetSpacial ? 0 : 20}
+           onMouseDown={() => {
+                  if (!isSearchPanel) setIsHovered(false);
+                  else setIsHovered(true);
```

Comment:
Isn't this `setIsHovered(!isSearchPanel);` ?

Reply:
Yes you're right, I can make it simpler, it would be `setIsHovered(isSearchPanel)` (whatever the value of `isSearchPanel` is)